assignment-tool/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -11690,9 +11688,9 @@ private void initializeAssignment(SessionState state)
 			try {
 				st = SiteService.getSite((String) state.getAttribute(STATE_CONTEXT_STRING));
 				if (contentreviewSiteMin > 0 && st.getTitle().length() < contentreviewSiteMin){
-					addAlert(state, rb.getFormattedMessage("review.sitechars", new Object[]{contentreviewSiteMin, reviewServiceName}));
+					addAlert(state, rb.getFormattedMessage("review.sitechars", new Object[]{reviewServiceName, contentreviewSiteMin}));
```

Comment:
Think that bit of code it's a bit before and doesn't appear on the patch.

2016-04-07 16:37 GMT+02:00 Matthew Buckett <notifications@github.com>:

> In
> assignment-tool/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java
> <https://github.com/ox-it/wl-assignment/pull/31#discussion_r58882140>:
>
> > @@ -11690,9 +11688,9 @@ private void initializeAssignment(SessionState state)
> >  			try {
> >  				st = SiteService.getSite((String) state.getAttribute(STATE_CONTEXT_STRING));
> >  				if (contentreviewSiteMin > 0 && st.getTitle().length() < contentreviewSiteMin){
> > -					addAlert(state, rb.getFormattedMessage("review.sitechars", new Object[]{contentreviewSiteMin, reviewServiceName}));
> > +					addAlert(state, rb.getFormattedMessage("review.sitechars", new Object[]{reviewServiceName, contentreviewSiteMin}));
>
> But you always check, even if the assignment isn't using a review service,
> don't you only want to check when editing a assignment that has selected to
> use a review service?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ox-it/wl-assignment/pull/31/files/6c849e75a7fc9db451dc0fe971510d1afc09b3d4#r58882140>
>